io.catenax.planned_production_output/1.0.0/PlannedProductionOutput.ttl

```diff
@@ -0,0 +1,186 @@
+    samm:description "Material identifier as assigned by the customer. This material number identifies the material (as planned) in the customer's database. For standard parts (e.g. screws) for which a customer does not have an own number, the material number supplier should be set in this field, too."@en;
+    samm:characteristic :MaterialNumber;
+    samm:exampleValue "MNR-7307-AU340474.002".
+:materialNumberCatenaX a samm:Property;
+    samm:preferredName "Material UUID Used in Catena-X"@en;
+    samm:description "Material identifier used uniquely within Catena-X to identify the material (e.g. PartAsPlanned)."@en;
+    samm:characteristic :CatenaXIdTraitCharacteristic;
+    samm:exampleValue "urn:uuid:48878d48-6f1d-47f5-8ded-a441d0d879df".
+:materialNumberSupplier a samm:Property;
+    samm:preferredName "Supplier Material Number"@en;
+    samm:description "Material identifier as assigned by the supplier. This material number identifies the material (as planned) in the supplier's database."@en;
+    samm:characteristic :MaterialNumber;
+    samm:exampleValue "MNR-8101-ID146955.001".
+:PositionsCollection a samm-c:Collection;
+    samm:preferredName "Positions Collection"@en;
+    samm:description "If any production for the customer is planned, then there is at least one position. If the supplier does differentiate the planned production according to the order positions (an OrderPositionReference is set), then multiple positions can be given. Among the positions, there could be only one position without an OrderPositionReference to either report the planned production independently of orders or to report internal stock orders planned for the customer."@en;
+    samm:dataType :Position.
+:MaterialNumber a samm:Characteristic;
+    samm:preferredName "Material Number"@en;
+    samm:description "The material number is a multi-character string, usually assigned by an ERP system."@en;
+    samm:dataType xsd:string.
+:CatenaXIdTraitCharacteristic a samm-c:Trait;
+    samm:preferredName "Catena-X ID Trait"@en;
+    samm:description "Trait to ensure UUID v4 data format."@en;
+    samm-c:baseCharacteristic :Uuidv4Characteristic;
+    samm-c:constraint :Uuidv4RegularExpression.
+:Position a samm:Entity;
+    samm:preferredName "Position"@en;
+    samm:description "The Position can be planned for production at several sites. A position may be anonymous or may reference an position within an order."@en;
+    samm:properties ([
+  samm:property :orderPositionReference;
+  samm:optional "true"^^xsd:boolean
+] :lastUpdatedOnDateTime :allocatedPlannedProductionOutputs).
+:Uuidv4Characteristic a samm:Characteristic;
+    samm:preferredName "UUID v4"@en;
+    samm:description "A version 4 UUID is a universally unique identifier that is generated using random 32 hexadecimal characters."@en;
+    samm:dataType xsd:string;
+    samm:see <https://tools.ietf.org/html/rfc4122>.
+:Uuidv4RegularExpression a samm-c:RegularExpressionConstraint;
```

Comment:
this is now probably the 10th or so model with a uuid characteristic definition. Maybe we should extract this to a separate shared aspect and reused in the other models?

Reply:
replaced by MS3 shared model. I'll rebase the branch after merging the model.